app/views/admin/roles/index.html.erb

```diff
@@ -23,53 +23,52 @@
     {
       text: "Organistions"
     },
-    {
-      text: "Role type"
-    },
     {
       text: "Currently appointed"
     },
     {
       text: "Translations"
     },
     {
-      text: "Edit"
+      text: tag.span(class: "govuk-visually-hidden")
     },
     {
-      text: "Delete"
-    }
+      text: tag.span(class: "govuk-visually-hidden")
+    },
   ],
   rows: @roles.map do |role|
     [
       {
-        text: role.name
+        text: tag.span(role.name, class: "govuk-!-font-weight-bold"),
       },
       {
         text: role.organisation_names
       },
       {
-        text: role.role_type.humanize
-      },
-      {
-        text: role.current_person_name
+        text:
+          if role.current_person_name.eql?("No one is assigned to this role")
+            ""
+          else
+            role.current_person_name
+          end
       },
       {
         text:
           content_tag_for(:div, role) do
-            link_to(sanitize("Manage translations#{tag.span(role.name, class: "govuk-visually-hidden")}"), admin_role_translations_path(role), title: "Manage translations of #{role.name}", class: "govuk-link")
+            link_to(sanitize("Manage#{tag.span(role.name, class: "govuk-visually-hidden")}"), admin_role_translations_path(role), title: "Manage translations of #{role.name}", class: "govuk-link")
```

Comment:
sorry i think we should leave a space between manage and `role.name` otherwise it will read it as one word 

```
"Manage #{tag.span(role.name, class: "govuk-visually-hidden")}"
```